src/_P184_Triac.ino

```diff
@@ -0,0 +1,380 @@
+      // dev.Ports              = 0;                                // Port to use when device has multiple I/O pins  (N.B. not used much)
+      dev.ValueCount         = 2;                                // The number of output values of a plugin. The value should match the number of keys PLUGIN_VALUENAME1_184
+      dev.OutputDataType     = Output_Data_type_t::Simple;      // Subset of selectable output data types  (Default = no selection)
+      dev.PullUpOption       = false;                            // Allow to set internal pull-up resistors.
+      dev.InverseLogicOption = false;                            // Allow to invert the boolean state (e.g. a switch)
+      dev.FormulaOption      = false;                            // Allow to enter a formula to convert values during read. (not possible with Custom enabled)
+      dev.Custom             = false;
+      dev.SendDataOption     = true;                            // Allow to send data to a controller.
+      dev.GlobalSyncOption   = true;                             // No longer used. Was used for ESPeasy values sync between nodes
+      dev.TimerOption        = true;                             // Allow to set the "Interval" timer for the plugin.
+      dev.TimerOptional      = false;                            // When taskdevice timer is not set and not optional, use default "Interval" delay (Settings.Delay)
+      dev.DecimalsOnly       = true;                             // Allow to set the number of decimals (otherwise treated a 0 decimals)
+      dev.CustomVTypeVar     = false;                            // Enable to allow the user to configure the Sensor_VType per Value that's available for the plugin
+      break;
+    }
+
+    case PLUGIN_GET_DEVICENAME:
+    {
+      // return the device name
+      string = F(PLUGIN_NAME_184);
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEVALUENAMES:
+    {
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_184));
+      strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[1], PSTR(PLUGIN_VALUENAME2_184));
+      break;
+    }
+
+    case PLUGIN_WEBFORM_SHOW_CONFIG:
+    {
+      // Called to show non default pin assignment or addresses like for plugins using serial or 1-Wire
+      // string += serialHelper_getSerialTypeLabel(event);
+      success = true;
+      break;
+    }
+
+    case PLUGIN_GET_DEVICEVALUECOUNT:
+    {
+      // This is only called when dev.OutputDataType is not Output_Data_type_t::Default
+      // The position in the config parameters used in this example is PCONFIG(P184_OUTPUT_TYPE_INDEX)
+      // Must match the one used in case PLUGIN_GET_DEVICEVTYPE  (best to use a define for it)
+      // see P026_Sysinfo.ino for more examples.
+      event->Par1 = 2; //getValueCountFromSensorType(static_cast<Sensor_VType>(PCONFIG(P184_OUTPUT_TYPE_INDEX)));
+      success     = true;
+      break;
+    }
```

Comment:
This case is useful when dynamically changing the number of device values, that's not used in this plugin, so this code can be removed.

Reply:
To be honest. I can't understand how to use this field.